sonarts-core/tests/rules/parametersMaxNumberRule/parametersMaxNumberRule.lint.ts

```diff
@@ -4,3 +4,35 @@
 }
 
 function ok(p1: number, p2: number, p3: number, p4: number, p5: number, p6: number, p7: number) { }
+
+
+class Foo {
+  // OK, class constructor with parameter properties are excluded
+  public constructor(
```

Comment:
how about using "independent" names and types? `p1`, `p2` ... , `number`, `string`....